libpromises/rlist.c

```diff
@@ -542,6 +542,15 @@ Rlist *RlistAppendScalar(Rlist **start, const char *scalar)
     return RlistAppendRval(start, RvalCopyScalar((Rval) { (char *)scalar, RVAL_TYPE_SCALAR }));
 }
 
+Rlist *RlistAppendString(Rlist **start, const char *string)
```

Comment:
[This does not represent a user facing change, and thus it should not have a changelog entry](https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#changelog). Please update commit message.